spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -7196,7 +7195,7 @@ developer could create the following TLD:
         "http://java.sun.com/xml/ns/j2ee web jsptaglibrary_2_0.xsd"
     version="2.0">
     <description>
-        Validates JSP pages to prohibit use of scripting elements.
+        Validates Jakarta Server Pages pages to prohibit use of scripting elements.
```

Comment:
Would just stating "Validates Jakarta Server Pages" be better than "Jakarta Server Pages pages"?